app/src/main/res/layout/fragment_login.xml

```diff
@@ -2,6 +2,12 @@
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
+    <data>
+        <variable
+            name="loginViewModel"
```

Comment:
It's not a good practice to use a viewModel inside an xml. If you want to set via xml the fields, you should pass something more dummy, like a data class.